src/ngraph/op/op_version_tbl.hpp

```diff
@@ -153,6 +153,7 @@ NGRAPH_OP(Multiply, ngraph::op::v0, 0)
 NGRAPH_OP(Multiply, ngraph::op::v1, 1)
 NGRAPH_OP(Negative, ngraph::op::v0, 0)
 NGRAPH_OP(NonMaxSuppression, ngraph::op::v1, 1)
+NGRAPH_OP(NonZero, ngraph::op::v3, 0)
```

Comment:
```suggestion
NGRAPH_OP(NonZero, ngraph::op::v3, 3)
```
If this is in the v3 namespace then it is a member of opset 3, not opset 0